Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -348,4 +349,12 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
   @objc fileprivate func saveButtonTapped(_ button: UIButton) {
     self.watchProjectViewModel.inputs.saveButtonTapped(selected: button.isSelected)
   }
+
+  func roundImage(_ url: URL) -> UIImage? {
+    guard let imageData = try? Data(contentsOf: url) else { return nil }
+
+    let avatar = UIImage(data: imageData, scale: UIScreen.main.scale)?
+      .af_imageRoundedIntoCircle()
```

Comment:
Since `AF` uses `rounded` naming would it make sense to also name our function `roundedImage` instead of `roundImage`?

Reply:
I agree, going to correct! thank you!

Reply:
If we look at our convenience func `ksr_setImageWithURL`, it calls `af_setImage` which accepts a `filter` argument. I believe in here, we could pass `CircleFilter()` which comes for free out of the box with Alamofire and wouldn't require us to convert the image url into `Data` before rounding it. One thing we could do is extend the functionality of `ksr_setImageWithURL` to accept an additional optional argument, maybe something like: `ksr_setImageWithURL(_ url: URL, rounded: Bool = false)` and passing the `CircleFilter()` into Alamofire's `filter` argument if `rounded` is `true`.

Another thing to note is that calling `Data(contentsOf: url)` is not recommended for remote url's because it's synchronous, which would [block the main thread](https://developer.apple.com/documentation/foundation/nsdata/1413892-init).